### DISCUSSION TITLE
How to disable pop-ups

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#16567] [How to]
Hi there

Need to disable popups as per attached screenshot, otherwise my colleagues will throw stones on me. Here is a lot of clicks in details view and this popups covering fields. Where to disarm this feature?

![image](https://user-images.githubusercontent.com/34973571/194744905-9628c049-29e5-49b7-85f0-5a3d08b32459.png)

**Comment by rskrzypczak** [comment]
Get a shield as these pop-ups are not configurable yet :)
